docs/podman-run.1.md

```diff
@@ -1087,8 +1092,8 @@ $ podman run -v /var/db:/data1 -i -t fedora bash
 ```
 
 Using --mount flags, To mount a host directory as a container folder, specify
-the absolute path to the directory and the absolute path for the container
-directory:
+the absolute path or volume name to the directory and the absolute path for 
```

Comment:
I think this should be "the absolute path to the directory or the volume name,".